tests/Detector.Tests/DotNetCore/RootDirectoryProjectFileProviderTest.cs

```diff
@@ -4,11 +4,10 @@
 // --------------------------------------------------------------------------------------------
 
 using System.IO;
-using Microsoft.Oryx.Detector.DotNetCore;
 using Microsoft.Oryx.Tests.Common;
 using Xunit;
 
-namespace Microsoft.Oryx.Detector.Tests.DotNetCore
+namespace Microsoft.Oryx.Detector.DotNetCore
```

Comment:
still want to keep it as Microsoft.Oryx.Detector.Test.DotNetCore, same as other test files' namespace